var/spack/repos/builtin/packages/mvapich2x/package.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+from spack import *
+
+class Mvapich2x(AutotoolsPackage):
+    # Mvapich2x is not installable from source and is only available through the binary mirror below
+    # http://mvapich.cse.ohio-state.edu:8080/download/mvapich/spack-mirror/mvapich2x/ 
+
+    homepage = "http://mvapich.cse.ohio-state.edu"
+    url      = "http://mvapich.cse.ohio-state.edu:8080/download/mvapich/spack-mirror/mvapich2x/mvapich2x-2.3.tar.gz"
+    
+    version('2.3', sha256='fc47070e2e9fac09b97022be2320200d732a0a4a820a2b51532b88f8ded14536')
+    version('2.3rc3', sha256='85a9f1ea1a837d487e356f021ef6f3a4661ad270a0c5f54777b362ee4d45166f')
```

Comment:
The way that Spack's version sorting works, 2.3rc3 is actually newer than 2.3 and will be installed by default. Either drop 2.3rc3 if you don't need it, or do:
```suggestion
    version('2.3', sha256='fc47070e2e9fac09b97022be2320200d732a0a4a820a2b51532b88f8ded14536', preferred=True)
    version('2.3rc3', sha256='85a9f1ea1a837d487e356f021ef6f3a4661ad270a0c5f54777b362ee4d45166f')
```